docs/sources/writing-guide/references/index.md

```diff
@@ -53,37 +70,39 @@ docs
 
 ```
-/docs/technical-documentation/branch/
-/docs/technical-documentation/branch/other/
-/docs/technical-documentation/leaf/
+/docs/writers-toolkit/branch/
+/docs/writers-toolkit/branch/other/
+/docs/writers-toolkit/leaf/
 ```
 
-All of the URL path based relrefs between these pages are are follows:
-
-| Source page                                   | Destination page                              | relref                                 |
-| --------------------------------------------- | --------------------------------------------- | -------------------------------------- |
-| `/docs/technical-documentation/branch/`       | `/docs/technical-documentation/branch/other/` | `{{</* relref "other" */>}}`           |
-| `/docs/technical-documentation/branch/`       | `/docs/technical-documentation/leaf/`         | `{{</* relref "../leaf" */>}}`         |
-| `/docs/technical-documentation/leaf/`         | `/docs/technical-documentation/branch/`       | `{{</* relref "../branch" */>}}`       |
-| `/docs/technical-documentation/leaf/`         | `/docs/technical-documentation/branch/other/` | `{{</* relref "../branch/other" */>}}` |
-| `/docs/technical-documentation/branch/other/` | `/docs/technical-documentation/branch/`       | `{{</* relref "./" */>}}`              |
-| `/docs/technical-documentation/branch/other/` | `/docs/technical-documentation/leaf/`         | `{{</* relref "../leaf" */>}}`         |
+To link to the source files directly, refer to the following table:
 
-You can refer to the table below for all file path based relrefs between these files.
-
-| Source file                      | Destination file                 | relref                                     |
+| Source page                      | Destination page                 | relref                                     |
 | -------------------------------- | -------------------------------- | ------------------------------------------ |
 | `/docs/sources/branch/_index.md` | `/docs/sources/branch/other.md`  | `{{</* relref "other.md" */>}}`            |
 | `/docs/sources/branch/_index.md` | `/docs/sources/leaf/index.md`    | `{{</* relref "../leaf/index.md" */>}}`    |
 | `/docs/sources/leaf/index.md`    | `/docs/sources/branch/_index.md` | `{{</* relref "../branch/_index.md" */>}}` |
 | `/docs/sources/leaf/index.md`    | `/docs/sources/branch/other.md`  | `{{</* relref "../branch/other.md" */>}}`  |
-| `/docs/sources/branch/other.md`  | `/docs/sources/branch/_index.md` | `{{</* relref "_index.md" */>}}`           |
+| `/docs/sources/branch/other.md`  | `/docs/sources/branch/_index.md` | `{{</* relref "./_index.md" */>}}`         |
 | `/docs/sources/branch/other.md`  | `/docs/sources/leaf/index.md`    | `{{</* relref "../leaf/index.md" */>}}`    |
 
-## Anchors
+To avoid having to worry about whether the destination is a leaf bundle, a branch bundle, or a regular file, you can omit the `/_index.md`, `/index.md` or `.md` extensions to have and the relref resolves the same.
```

Comment:
Serial commas. 
```suggestion
To avoid having to worry about whether the destination is a leaf bundle, a branch bundle, or a regular file, you can omit the `/_index.md`, `/index.md`, or `.md` extensions to have and the relref resolves the same.
```

Reply:
Good catch. This should say:

```suggestion
To avoid having to worry about whether the destination is a leaf bundle, a branch bundle, or a regular file, you can omit the `/_index.md`, `/index.md`, or `.md` extension.
In each case, the relref resolves to the same page.
```